p2p/pex/addrbook.go

```diff
@@ -550,8 +537,7 @@ func (a *addrBook) getBucket(bucketType byte, bucketIdx int) map[string]*knownAd
 	case bucketTypeOld:
 		return a.bucketsOld[bucketIdx]
 	default:
-		cmn.PanicSanity("Should not happen")
-		return nil
+		panic("Should not happen")
```

Comment:
Can we change this to: 
```suggestion
		panic("Invalid bucket type")
```